### PR TITLE
A few minor copyedits

### PR DESCRIPTION
**Pull request by mlncn**
Hi!  Long time no talk!

Good grief there's a typo in my commit messages— that's an added **colon**, not a semicolon.

### CODE REVIEW

File: book-cw/part-4-managing/complexity.adoc [hunk 1]
```diff
@@ -560,7 +560,8 @@ settings:
 === Flat-file shortcomings
 
 Remember that concept of _relationality_ in small data that I brought up earlier?
-Where
+Where sets of structured data need to be related to one another, flat files fall short.
```
Comment (briandominick) [inline book-cw/part-4-managing/complexity.adoc]: Love it. Well done. You always were able to finish my sentences better ;-)

**Comment by briandominick** [comment]
Crap I regret merging this without making you [add your name to the contributors file](https://github.com/briandominick/codewriting#editorial-process).

**Comment by mlncn** [comment]
I did read that section!  But i'll have to correct more than a typo or two to earn it.  Unfortunately, although i returned to your book in deep frustration with ReadTheDocs in my attempt to allow people to contribute in Markdown while using ReStructuredText for structuring the documentation, the reasons for seeking the lowest common denominator by using ReadTheDocs while downgrading to Markdown remain (and it looks like i will in fact be able to get what i was trying to get).  The reasons are that the documentation needs to be written largely by volunteers, not professionals that can set up toolchains.

The main project is [Drutopia](https://drutopia.org/) - docs at http://docs.drutopia.org

I'm thinking hard about how to keep documentation written for specific features *with* the repositories for those features and powering both the full docs site and also in-place documentation— OK, i'm not thinking hard, i'm keeping it in the back of my mind and hoping something clicks eventually.  Keeping documentation for, say, [Drutopia Action](https://gitlab.com/drutopia/drutopia_action) in that repository while including it at docs.drutopia.org also would probably necessitate moving away from ReadTheDocs unless maybe i could do something clever with submodules.  But the more powerful thing *could* be integration of contributor/developer, administrator, editor, and other end user documentation into one system.  But i've got no vision for that now.

We want to make things super easy to use, but really easy to contribute to and influence also.  So being able to rather seamlessly dive ever deeper into documentation would be awesome.

**Comment by briandominick** [comment]
The Drutopia docs look great. I don't know what all your requirements are going to be in the long run, but RtD is suitable for a huge share of documentation cases. I think my entire career now revolves around a relative minority of cases, within the broad space of software docs. And I also suspect WtD is a complete system, at least if you're using reStructuredText, where any shortcomings of the RtD framework itself can be handled by preprocessing elements of the source you feed Sphinx. I would like to explore that toolchain someday. It seems like it must be hard to theme or something, since everyone goes with the generic. Though that's largely true of AsciiDoc project, as well. I'll dig around some more when I get a chance and let you know what I think. Cheers!

**Pull request merged**